frontend/src/components/Navbar.jsx

```diff
@@ -0,0 +1,52 @@
+import React, { useState } from "react";
+
+function Navbar() {
+  const [isActive, setIsActive] = useState(false);
+
+  const handleIconClick = () => {
+    setIsActive(!isActive);
+  };
+
+  const handleLinkClick = () => {
+    setIsActive(false);
+  };
+
+  return (
+    <div>
```

Comment:
Tous passer en Link avec react router dom et l'attribut to
Pas de onClick sur un a (pb accessibilité)